roles/lib_utils/library/repoquery.py

```diff
@@ -477,7 +477,7 @@ def format_versions(self, formatted_versions):
         ''' Gather and present the versions of each package '''
 
         versions_dict = {}
-        versions_dict['available_versions_full'] = formatted_versions.keys()
+        versions_dict['available_versions_full'] = list(formatted_versions.keys())
```

Comment:
@detiber @kwoodson I see I missed to update this same line of code in `roles/lib_utils/src/class/repoquery.py`, and CI did not fail... we possibly have a problem with the code generation verification.